### DISCUSSION TITLE
Welcome to django-wiki Discussions!

### DISCUSSION THREAD

**Discussion by benjaoming** [django-wiki/django-wiki#1150] [Announcements]
## 👋 Welcome!

Please:

* use this space instead of opening issues, if you have questions
* ..or if you have ideas
* want to say hi to other users
* welcome others and be open-minded. Remember that this is a community we build together.

Please also see existing questions on Stack Overflow: https://stackoverflow.com/search?q=django-wiki

**Comment by rcooke** [comment]
Hi there!  I have been using TRAC for a very long time. But since they are still stuck with Py27 I decided to try this out for a new project and client.

Its been great so far, but, I would appreciate more info about setup - especially plugins.  In TRAC its the plugins that make it really rock.

Another important area is security, I am looking for info on how to lock down the wiki so it can only be accessed by staff people. Although having a public page is fine, I need to ensure public cannot access or even see anything else on the site.
